README.md

```diff
@@ -2,6 +2,93 @@
 
 Build images from the commandline in a convenient way.
 
+## Installation
+
+This project is under development right now and needs to be run via:
+```
+$ go run github.com/osbuild/image-builder-cli/cmd/image-builder@main
+```
+we plan to provide rpm packages as well.
+
+
+## Prerequisites
+
+Make sure to have the required `osbuild` rpms installed:
+```
+$ dnf install osbuild osbuild-depsolve-dnf
```

Comment:
Nitpick: `dnf install` needs to be run as root, so I suggest using either of these options:

```suggestion
# dnf install osbuild osbuild-depsolve-dnf
```

```suggestion
$ sudo dnf install osbuild osbuild-depsolve-dnf
```